mix/Extension.cpp

```diff
@@ -17,13 +17,14 @@
  * Ethereum IDE client.
  */
 
+#include "AppContext.h"
```

Comment:
This was actually reordering for the same reasons asio.h was being included. To ensure boost includes come before windows.h